test/index.js

```diff
@@ -145,6 +155,24 @@ describe('simple-keyring', function () {
         expect(adr).toBe(accountAddress);
       });
     });
+
+    it('throw error for invalid message', async function () {
+      await expect(keyring.signMessage(address, '')).rejects.toThrow(
+        'Expected message to be an Uint8Array with length 32',
+      );
+    });
+
+    it('throw error if empty address is passed', async function () {
+      await expect(keyring.signMessage('', message)).rejects.toThrow(
+        'Must specify address.',
+      );
+    });
+
+    it('throw error if wrong address is passed', async function () {
+      await expect(
+        keyring.signMessage(invalidAddress, message),
```

Comment:
[nit] - calling this `invalidAddress` makes it sound like its not a valid ethereum address when really its just a random address not associated with the keyring. Maybe we should call it `notKeyringAddress` or something like that?

Reply:
Good point, I updated it.